gui/mozregui/wizard.py

```diff
@@ -111,6 +111,27 @@ def __init__(self):
         self.ui.start_date.setDateTime(now.addYears(-1))
         self.ui.end_date.setDateTime(now)
 
+    def validatePage(self):
+        good_date = self.ui.start_date.date()
+        bad_date = self.ui.end_date.date()
+        current = QDateTime.currentDateTime().date()
+        if good_date < bad_date:
+            if bad_date <= current:
+                return True
+            else:
+                QMessageBox.critical(
+                    self,
+                    "Error",
+                    "Date you entered is larger than now,please try it again.")
+                return False
+        else:
+            QMessageBox.critical(
+                self,
+                "Error",
+                "Dates in good date field can't larger than bad date,\
+                please try it again.")
```

Comment:
Same thing for the "please try", not sure it helps. Also we should avoid to use the good / bad words here (since user may have checked "Search fix"). Maybe "The first date must be earlier than the second one." ?

Last thing to note, When a line with a string is too long, please use something like here: https://github.com/mozilla/mozregression/blob/master/mozregression/main.py#L93. The syntax you used is not equivalent, as it add spaces into the string.